src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/AuthorizationServer/ApiAccess/edit.html.twig

```diff
@@ -23,10 +23,20 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
 
-{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% extends '@PrestaShop/Admin/Configure/AdvancedParameters/AuthorizationServer/ApiAccess/form.html.twig' %}
 
-{% block content %}
-    {% include '@PrestaShop/Admin/Configure/AdvancedParameters/AuthorizationServer/ApiAccess/form.html.twig' with {
-      apiAccessForm: apiAccessForm
-    } %}
+{% block form_submit_buttons %}
+  <a
+    class="btn btn-primary generate-client-secret"
+    href="{{ path('admin_api_accesses_regenerate_secret', {apiAccessId: app.request.get('apiAccessId')}) }}"
+    data-confirm-title="{{ 'Regenerate client secret'|trans({}, 'Admin.Advparameters.Feature') }}"
+    data-confirm-message="{{ 'If you regenerate your client secret your former secret will become obsolete.'|trans({}, 'Admin.Advparameters.Feature') }}"
```

Comment:
```suggestion
    data-confirm-message="{{ 'If you regenerate your client secret, your former client secret will become obsolete.'|trans({}, 'Admin.Advparameters.Feature') }}"
```

@jolelievre It looks like a help text. If yes, it should be in Admin.Advparameters.Help

Reply:
Actually, it's a warning modal message:

![Capture d’écran 2023-10-27 à 10 33 44](https://github.com/PrestaShop/PrestaShop/assets/13801017/00675350-c7a4-4dd1-bbd4-11715031cecd)

Reply:
Ok then let's keep it in Admin.Advparameters.Feature, thank you! 

Could have "Cancel" instead of "Close"?